configs/templates/hpc-stack-dev/spack.yaml

```diff
@@ -192,21 +192,21 @@ spack:
     udunits:
       version: [2.2.28]
     upp:
-      version: [10.0.10]
+      version: [10.0.9]
     w3emc:
       version: [2.9.2]
     w3nco:
       version: [2.4.1]
     wget:
       version: [1.21.2]
     wgrib2:
-      version: [3.1.1]
+      version: [2.0.8]
     wrf-io:
       version: [1.2.0]
     yafyaml:
       version: [0.5.1]
     zlib:
-      version: [1.2.12]
+      version: [1.2.11]
```

Comment:
This will require adding a flag to the spack config to be able to install deprecated packages.

Reply:
See here: https://spack.readthedocs.io/en/latest/config_yaml.html#config-yaml

Reply:
Would it make sense to add "deprecated: true" to configs/common/config.yaml and/or configs/sites/*/config.yaml?

Reply:
I think it is best to do that in each of the templates that needs this flag. The reason for having the flag in the first place is that deprecating versions is often done because of security concerns etc. We want to make sure to only use it where needed, I assume.

Reply:
That makes sense, though I don't immediately see how to do this in a template-specific way because it is set under config.yaml, not spack.yaml. Maybe it would make sense to decide on a site-by-site basis whether we want to allow deprecated packages to be used and update configs/sites/*/config.yaml accordingly?

Reply:
Ah yep, sure enough, I must have popped it into the wrong section before. I just committed the updated template file.